addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -470,18 +466,23 @@ def _import_fattura_pa(self, tree, invoice):
                                             invoice_line_form.product_id = product
                                             break
                                     if partner:
-                                        product_supplier = self.env['product.supplierinfo'].search([('partner_id', '=', partner.id), ('product_code', '=', code.text)], limit=1)
-                                        if product_supplier and product_supplier.product_id:
+                                        product_supplier = self.env['product.supplierinfo'].search([('name', '=', partner.id), ('product_code', '=', code.text)], limit=2)
```

Comment:
@dbkosky How could this work?

Reply:
@jco-odoo I just changed it to match the original pr, there was a git conflict here

Reply:
@dbkosky but how is the field called in saas-15.1? name or partner_id?

Reply:
@jco-odoo you're right